### PR TITLE
created details popup html,css and js

### PR DESCRIPTION
**Pull request by shayan1234554321**
In this project, I and my partner @jaimevillegas worked together to achieve a details popup window. We added the followings :
- Added details popup HTML 
- Added styling using CSS
- Added functionality using JS
- Added icons for popup

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -252,6 +269,81 @@
...
+    width: 100%;
+    height: 100%;
+    background-color: #c1c7d065;
+    backdrop-filter: blur(6px);
+    z-index: 30;
+    display: none;
+    align-items: center;
+    justify-content: center;
+  }
+
```
Comment (BregornOriginal) [inline index.css:286]: - [ ] Good work creating the popup window 💯 . Kindly add the grey line between buttons and skills missing on your popup window
- [ ] Kindly align your buttons in rows as in the FIGMA design

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @Team 💯 💯 💯  

## Good job so far!

## You still need to work on some issues to go to the next project, but you are almost there!

one-step-closer-one-more

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good pull request description
✅ Excelent work implementing a dynamic popup window! 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css (hunk 1 above)
Comment (shayan1234554321) [reply index.css:286]: Thanks for the great review 😍. I did implement this design, but still will be causing this effect for more readability ✍️ in extremely smaller screens like this one  ( image attached ) , but solved in the other screen sizes ✌️. 
image

**Review by gemmen29 — APPROVED** [review]
Hi @shayan1234554321, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**